src/products/views/ProductVariant.tsx

```diff
@@ -35,6 +35,10 @@ export const ProductVariant: React.FC<ProductUpdateProps> = ({
   const navigate = useNavigator();
   const notify = useNotifier();
   const intl = useIntl();
+  const [errors, setErrors] = useState([]);
+  useEffect(() => {
+    setErrors([]);
+  }, [variantId]);
```

Comment:
How about `useEffect(() => setErrors([]), [variantId]);`?
#285 will autofix this but it's not merged yet 🤷‍♂️